src/components/views/dialogs/BaseDialog.js

```diff
@@ -45,19 +43,30 @@ export default React.createClass({
 
         // children should be the content of the dialog
         children: React.PropTypes.node,
+
+        // Id of content element
+        // If provided, this is used to add a aria-describedby attribute
+        contentId: React.PropTypes.string
+    },
+
+    componentDidMount: function() {
+        this.applicationNode = document.getElementById('matrixchat');
```

Comment:
Perhaps add a comment here that this is to hide the rest of the application from screen readers while the dialog is open

Reply:
Done in 4171675